notebooks/MAST/m_dwarf_flares_hst/m_dwarf_flares_hst.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<p>Is the .csv basically a table from a publication? If so you could use Vizier. I do this in a tutorial in PR 12 if you want to see an example. You probably wouldn't need to go into as much detail. Alternatively you could just cite the paper and describe how you got the table and just include the .csv if it would otherwise lose focus from the tutorial.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>